apps/system/style/modal_dialog/modal_dialog.css

```diff
@@ -85,6 +85,7 @@
 
 #modal-dialog-select-one h3 {
   font-size: 1.8rem;
+  line-height: 2rem;
 }
 
 #modal-dialog-select-one ul {
```

Comment:
The button text is badly placed. Please add the following rules to `#modal-dialog-select-one ul > li > button`:

    padding-left: 1.5rem;
    line-height: 2rem;